lang_GENERIC/analyze/AST_to_IL.ml

```diff
@@ -71,16 +71,40 @@ let sgrep_construct any_generic =
   error_any any_generic "Sgrep Construct"
 (*e: function [[AST_to_IL.sgrep_construct]] *)
 
-(*s: function [[AST_to_IL.todo]] *)
-let todo any_generic =
-  error_any any_generic "TODO Construct"
-(*e: function [[AST_to_IL.todo]] *)
-
 (*s: function [[AST_to_IL.impossible]] *)
 let impossible any_generic =
   error_any any_generic "Impossible Construct"
 (*e: function [[AST_to_IL.impossible]] *)
 
+exception Todo of G.any
+
+(*s: function [[AST_to_IL.todo]] *)
```

Comment:
Please do not modify or add those comments that start with s:, e:, or x:. They are added and used
by my literate programming tool syncweb, and modifying or adding new one will confure the tool.
Just ignore them. Even if you rename a function, do not modify the related comment.

Reply:
Should I remove those `s:` and `e:` comments together with the previous definition of `todo`?

Reply:
I removed the comments.